src/queryClient.ts

```diff
@@ -48,10 +49,24 @@ const retry = (failureCount: number, error: Error): boolean => {
   return false;
 };
 
-export default (config: Partial<QueryClientConfig>) => {
+export type ConfigureQueryClientConfig = Partial<
+  Pick<
+    QueryClientConfig,
+    | 'API_HOST'
+    | 'DOMAIN'
+    | 'SHOW_NOTIFICATIONS'
+    | 'onConfigAxios'
+    | 'enableWebsocket'
+    | 'WS_HOST'
+    | 'notifier'
+  >
+> &
+  Pick<QueryClientConfig, 'defaultQueryOptions'>;
```

Comment:
What is the logic behind having all settings optional but the `defaultQueryConfig` mandatory ?